biztalk/core/requirements-phase-recommendations.md

```diff
@@ -49,7 +49,7 @@ The primary deliverable associated with the requirements phase is a requirements
  For example, each transport adapter type has its own performance and scale characteristics. If the desired throughput exceeds the ability of one or more of the adapter types (either receive or send), then alternatives for scaling the adapter may need to be investigated.
 
 ## Estimate Sizing
- Based on the established goals and criteria, it is never too early to begin the process of estimating the hardware sizing that will be required to meet the goals. As with any sizing estimation efforts, one must base the estimates on actual test results. During the early phases of a project, those results must come from external sources. You can read case studies at BizTalk Server Developer Center, at [https://go.microsoft.com/fwlink/?LinkId=49339](https://go.microsoft.com/fwlink/?LinkId=49339). The case studies provide details about the scenarios tested, the hardware on which testing was done, and the configuration for the tests. You can extrapolate from the performance achieved for these test cases to get an initial sizing estimate for your system.
+ Based on the established goals and criteria, it is never too early to begin the process of estimating the hardware sizing that will be required to meet the goals. As with any sizing estimation efforts, one must base the estimates on actual test results. During the early phases of a project, those results must come from external sources. You can read case studies at BizTalk Server Developer Center. The case studies provide details about the scenarios tested, the hardware on which testing was done, and the configuration for the tests. You can extrapolate from the performance achieved for these test cases to get an initial sizing estimate for your system.
```

Comment:
The "BizTalk Server Developer Center" no longer exists, so replacing with something similar:

```suggestion
Based on the established goals and criteria, it is never too early to begin the process of estimating the hardware sizing that will be required to meet the goals. As with any sizing estimation efforts, one must base the estimates on actual test results. During the early phases of a project, those results must come from external sources. For examples and case studies, see [Security Case Studies for Small to Medium-Sized Companies](/biztalk/core/security-case-studies-for-small-to-medium-sized-companies).
```